game-state-service

```diff
@@ -145,6 +168,18 @@ class GameStateService(Gio.Application):
                     'key %s does not exist' % key)
             else:
                 invocation.return_value(GLib.Variant('(v)', (value,)))
+        elif method == 'Delete':
+            if self._state.has(key):
+                self._state.delete(key)
+                connection.emit_signal(None, path, iface, 'changed', params)
+            invocation.return_value(None)
+        elif method == 'DeleteAll':
+            current_keys = self._state.get_keys()
+            self._state.delete_all()
+            for current_key in current_keys:
+                connection.emit_signal(None, path, iface, 'changed',
+                                       GLib.Variant('(s)', (current_key,)))
```

Comment:
just double checking, but the expected behavior is that clients would receive the `changed` signal, and because of the fact that there ins't a value for the given key, it would be implicitly mean it was deleted?